packages/middleware/src/types/common.ts

```diff
@@ -107,6 +119,7 @@ export interface LoadInitConfigProps {
   apiClient: ApiClientFactory;
   integration: Integration;
   tag: string;
+  alokai: AlokaiContainer;
```

Comment:
```suggestion
  integrationContext: AlokaiContainer;
```
I am not sure ;/ this alokai thing is soooooooo bad ;/ So is the overusing the context name. And I think that there might be a different context already injected? I am not sure what are these all types for eh

Reply:
We've decided together, we will adjust it in next task